spinalcordtoolbox/vertebrae/core.py

```diff
@@ -497,45 +500,26 @@ def label_segmentation(fname_seg, list_disc_z, list_disc_value, verbose=1):
     seg.change_orientation(init_orientation).save(add_suffix(fname_seg, '_labeled'))
 
 
-def label_discs(fname_seg_labeled, verbose=1):
+def label_disc(fname_seg, list_disc_z, list_disc_value, verbose=1):
```

Comment:
why removing the "s" if you label more than 1 disc?

Reply:
done